atlp-rwanda/code-ninjas-be
# #181414705 (chore) CI testing with Github actions

### What does this PR do?
 Sets up continuous integration testing
### Description of Task to be completed?
Configure a Github workflow that runs test for every pull request into develop or main branches
### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - checkout to your new branch
  - Add your changes
  - Push to your new remote branch
  - Create a new pull request to develop
  - At the bottom of your pull request it should be running the tests
  - Await to see if the checks pass
 
### Any background context you want to provide?
Configured with coveralls. A badge on the README shall read the coverage of all the tests
### What are the relevant pivotal tracker stories?
[##181414705](https://www.pivotaltracker.com/story/show/181414705)
### Screenshots (if appropriate)
### Questions:
